pkg/app/master/command/images/handler.go

```diff
@@ -126,6 +128,11 @@ func OnCommand(
 	images, err := crtClient.ListImages(cparams.Filter)
 	xc.FailOn(err)
 
+	if cparams.TUI {
+		model := models.InitialImages(images)
+		tui.RunTUI(model)
+	}
```

Comment:
will be good to term this into `else if xc.Out.Quiet` with the next block of code that follows.